nextstrain/cli/runner/conda.py

```diff
@@ -516,31 +516,36 @@ def update() -> UpdateStatus:
 
     nextstrain_base = PackageSpec.parse(NEXTSTRAIN_BASE)
 
-    current_version = (package_meta(NEXTSTRAIN_BASE) or {}).get("version")
+    current_meta = package_meta(NEXTSTRAIN_BASE) or {}
+    current_version = current_meta.get("version")
+    current_subdir = current_meta.get("subdir") or platform_subdir()
 
-    if latest_dist := package_distribution(NEXTSTRAIN_CHANNEL, NEXTSTRAIN_BASE):
-        assert nextstrain_base.name == latest_dist.name
+    assert current_meta.get("name") in {nextstrain_base.name, None}
 
-        latest_version = latest_dist.version
+    if latest_dist := package_distribution(NEXTSTRAIN_CHANNEL, NEXTSTRAIN_BASE, current_subdir):
+        assert latest_dist.name == nextstrain_base.name
```

Comment:
Why the flipping of `nextstrain_base.name == latest_dist.name` to `latest_dist.name == nextstrain_base.name`?